command/recorderd/file_watcher_test.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"os"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/bitmark-inc/logger"
+	"github.com/fsnotify/fsnotify"
+)
+
+const (
+	testFileName = "testWatcher"
+)
+
+var (
+	removeChannel = make(chan struct{}, 1)
+	changeChannel = make(chan struct{}, 1)
+)
+
+func setupTestFileWatcher(t *testing.T) *FileWatcherData {
+	removeTestFiles()
+	setupLogger(t)
+	w, _ := fsnotify.NewWatcher()
```

Comment:
this is a test, everything is setup, so it won't be necessary to consider it

Reply:
In a test I would fatal error to display fsnotify error return

Reply:
i was thinking when watcher initialize fail, all tests will fail and then find out the problem.

but showing error message, it could show error message earlier